aclogview/CM_Magic.cs

```diff
@@ -127,8 +130,7 @@ public class RemoveSpell : Message {
 
         public override void contributeToTreeView(TreeView treeView) {
             TreeNode rootNode = new TreeNode(this.GetType().Name);
-            rootNode.Expand();
-            rootNode.Nodes.Add("i_spell_id = " + i_spell_id);
+            rootNode.Nodes.Add("i_spell_id = " + Utility.FormatGuid((uint)this.i_spell_id));
```

Comment:
You deleted the node expansion call above this. Please add it back - otherwise it's annoying to have to manually expand such simple nodes in the UI.